tools/travis-test.sh

```diff
@@ -101,8 +101,8 @@ setup_chroot()
 
   # install needed packages
   sudo chroot $DIR bash -c "apt-get install -qq -y \
-    libatlas-base-dev gfortran python-dev python-nose python-pip cython \
-    python-pytest"
+    libatlas-base-dev gfortran python-dev python3-dev python-pip python3-pip \
+    cython  python3-pytest python-pytest"
```

Comment:
Do we still need all the non-python3 versions if we are running Python3?

Reply:
possibly, but then we cannot run chroot python2 tests anymore. If that is the direction I will change this